.eleventy.js

```diff
@@ -64,7 +66,8 @@ const expandContributors = require(`./${filtersDir}/expand-contributors`);
 const githubLink = require(`./${filtersDir}/github-link`);
 const htmlDateString = require(`./${filtersDir}/html-date-string`);
 const md = require(`./${filtersDir}/md`);
-const paginate = require(`./${filtersDir}/paginate`);
+const pageContent = require(`./${filtersDir}/page-content`);
```

Comment:
what do you think about renaming this to `addPagination` to be a bit more explicit?